contrib/tools/stacks-relay/src/main.rs

```diff
@@ -0,0 +1,54 @@
+use std::{
+    io::{Read, Write},
+    net::TcpListener,
+    str::from_utf8,
+};
+
+use crate::{http::RequestMessageEx, state::State, url::QueryEx};
+
+mod http;
+mod state;
+mod url;
+
+fn main() {
+    let mut state = State::default();
+    let listner = TcpListener::bind("127.0.0.1:9776").unwrap();
+    for stream_or_error in listner.incoming() {
+        let mut stream = stream_or_error.unwrap();
```

Comment:
```suggestion
    for stream in listner.incoming().unwrap() {
```

Reply:
@netrome it will not work. `incoming()` is an iterator. `stream_or_error` is an item of the iterator.